qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -207,7 +214,7 @@ def _is_one_op_non_discrete(ops):
             if do_consolidate_blocks_init:
                 init.append(Collect2qBlocks())
                 init.append(ConsolidateBlocks())
-                init.append(Split2QUnitaries())
+                init.append(Split2QUnitaries(pass_manager_config.approximation_degree))
```

Comment:
This needs to handle the case of `None` correctly. This would be something like:

```suggestion
                # If approximation degree is None that indicates a request to approximate up to the
                # error rates in the target. However, in the init stage we don't yet know the target
                # qubits being used to figure out the fidelity so just use the default fidelity parameter
                # in this case.
                if pass_manager_config.approximation_degree is not None:
                    init.append(Split2QUnitaries(pass_manager_config.approximation_degree))
                else:
                    init.append(Split2QUnitaries())
```

Reply:
I wasn't aware it can be `None`, the default seems to be setting `approximate_degree=1`, thanks!